system/infra-monitoring/vendor/bm-cablecheck-exporter/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
```

Comment:
please add annotations similar to:
https://github.com/sapcc/helm-charts/blob/master/prometheus-exporters/vrops-exporter/templates/service.yaml#L6-L9